webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback } from 'react';
+
+import { Modal } from 'components/kit/Modal';
+import { makeToast } from 'components/kit/Toast';
+import { updateGroup } from 'services/api';
+import { V1GroupSearchResult, V1User } from 'services/api-ts-sdk';
+import handleError, { ErrorType } from 'utils/error';
+
+interface Props {
+  groupResult: V1GroupSearchResult;
+  user?: V1User;
+  fetchGroups: () => Promise<void>;
+  onExpand: (expand: boolean, record: V1GroupSearchResult) => void;
+}
+
+const RemoveUserFromGroupModalComponent = ({
+  groupResult,
+  user,
+  fetchGroups,
+  onExpand,
+}: Props): JSX.Element => {
+  const onRemoveUser = useCallback(
+    async (record: V1GroupSearchResult, userId?: number) => {
+      const {
+        group: { groupId },
+      } = record;
+      if (!groupId || !userId) return;
+      try {
+        await updateGroup({ groupId, removeUsers: [userId] });
+        makeToast({
+          severity: 'Confirm',
+          title: 'User has been removed from group.',
+        });
+        onExpand(true, record);
+        fetchGroups();
+      } catch (e) {
+        makeToast({ severity: 'Error', title: 'Error deleting group.' });
+        handleError(e, { silent: true, type: ErrorType.Input });
+      }
+    },
+    [onExpand, fetchGroups],
+  );
+
+  const handleOk = useCallback(() => {
+    onRemoveUser(groupResult, user?.id);
+  }, [groupResult, onRemoveUser, user]);
+
+  return (
+    <Modal
+      danger
+      size="small"
+      submit={{
+        handleError,
+        handler: handleOk,
+        text: 'Remote User',
+      }}
+      title="Confirm Delete User from Group">
+      <div>
+        Are you sure you want to remove `{user?.username ?? 'this user'}` from `
+        {groupResult.group.name ?? 'this group'}`?
```

Comment:
<img width="354" alt="Screenshot 2023-10-18 at 4 28 20 PM" src="https://github.com/determined-ai/determined/assets/40620519/3faff456-5894-440c-a390-5c5b87a090f5">
The appearance looks a bit odd to me, can we remove the ``? 
Also the title should be `Confirm Removing User from Group`?

Reply:
fixed